crt_portal/cts_forms/models.py

```diff
@@ -357,6 +358,13 @@ def contact_full_name(self):
         return first or last
 
 
+class ReportAttachment(models.Model):
+    file = models.FileField(upload_to='attachments', validators=[validate_file_attachment])
+    filename = models.CharField(max_length=255)
+    user = models.ForeignKey(User, blank=True, null=True, on_delete=models.SET_NULL)
+    report = models.ForeignKey(Report, on_delete=models.CASCADE, related_name='attachments')
+
```

Comment:
We're appending a datetime stamp to the file names as stored in S3, it may be good to track a `datetime_created` on the model as well.

Reply:
yes!